src/solvers/dgsem_tree/containers_2d.jl

```diff
@@ -1372,6 +1372,7 @@ function Base.resize!(container::ContainerSubcellLimiterIDP2D, capacity)
     @unpack _alpha, _alpha1, _alpha2 = container
     resize!(_alpha, n_nodes * n_nodes * capacity)
     container.alpha = unsafe_wrap(Array, pointer(_alpha), (n_nodes, n_nodes, capacity))
+    container.alpha .= zero(eltype(container.alpha))
```

Comment:
Why is the initialization necessary?

Reply:
It is not necessary. Nonetheless, I preferred this initialization to prevent non-initialized values in ParaView for the first solution file. See also https://github.com/trixi-framework/Trixi.jl/pull/1611#discussion_r1315573338

Reply:
If I were you, I'd consider to not use zero, which could indicate "not yet used" or "this is actually zero". What about a sentinel value such as `-1`? IIRC, `-1` is not a valid value such that whenever one encounters it, it is clear that this is not a valid alpha.

Reply:
Oh yes, that's a good idea. Thank you.

Reply:
Yes, alpha cannot be `-1`. It would also be possible to set the alphas to `NaN`, right? What is the normal preference in these cases? @sloede

Reply:
I just thought about this again and I think `NaN` would actually be nicer.
If we initialize the values with `-1`, rescale a later ParaView file (probably to `[0,1]`) and go back to the first file, it still looks like the values are `0` everywhere. With `NaN` this is not a problem.
Do you agree with that @sloede ?

Reply:
Sounds reasonable.